moveos/moveos-stdlib/moveos-stdlib/sources/account_storage.move

```diff
@@ -115,61 +74,37 @@ module moveos_std::account_storage {
         table::contains(&self.modules, name)
     }
 
-    // === Global storage functions ===
+    // === Account Storage Functions
+
+    public fun borrow_resource<T: key>(self: &AccountStorage): &T {
+        borrow_resource_from_account_storage<T>(self)
+    }
 
-    #[private_generics(T)]
-    /// Borrow a resource from the account's storage
-    /// This function equates to `borrow_global<T>(address)` instruction in Move
-    public fun global_borrow<T: key>(ctx: &Context, account: address): &T {
-        let account_storage = borrow_account_storage(ctx, account);
-        borrow_resource_from_account_storage<T>(account_storage)
+    public fun borrow_mut_resource<T: key>(self: &mut AccountStorage): &mut T {
```

Comment:
AbstractStorage has two types of storage, one is resource and the other is object. Do we need to consider naming objects similar to ObjectStorage?
Another question, StorageContext is almost unused in the framework now, will it be abandoned directly in the future?

Reply:
StorageContext is for supporting App Storage Context in the future. It is unused now and does not affect the Framework API.

Reply:
For Object, now all Object storage functions are implemented in the `object` module. Maybe we can migrate it into the `storage_context` or `object_storage` module, but these are all the implementation details. 

Account resource and object public API in the `context`, The refactor does not affect the developer.